Buffer.alloc dependencies breaking test

```
Failed tests:

  11) Felicity Example should return a buffer:

      Buffer.alloc is not a function

      at Object.internals.string.internals.number.internals.binary.schema [as binary] (/Users/lchan/Projects/felicity/lib/helpers.js:323:110)
      at Object.exampleGenerator [as example] (/Users/lchan/Projects/felicity/lib/exampleGenerator.js:16:138)
      at /Users/lchan/Projects/felicity/test/felicity_tests.js:92:34
      at Immediate._onImmediate (/Users/lchan/Projects/felicity/node_modules/lab/lib/runner.js:647:36)
      at processImmediate [as _immediateCallback] (timers.js:383:17)

  84) Binary should return a buffer:

      Buffer.alloc is not a function

      at Object.internals.string.internals.number.internals.binary.schema [as binary] (/Users/lchan/Projects/felicity/lib/helpers.js:323:110)
      at /Users/lchan/Projects/felicity/test/value_generator_tests.js:514:40
      at Immediate._onImmediate (/Users/lchan/Projects/felicity/node_modules/lab/lib/runner.js:647:36)
      at processImmediate [as _immediateCallback] (timers.js:383:17)

  85) Binary should return a string with specified encoding:

      Buffer.alloc is not a function

      at Object.internals.string.internals.number.internals.binary.schema [as binary] (/Users/lchan/Projects/felicity/lib/helpers.js:323:110)
      at /Users/lchan/Projects/felicity/test/value_generator_tests.js:534:44
      at Array.forEach (native)
      at /Users/lchan/Projects/felicity/test/value_generator_tests.js:531:28
      at Immediate._onImmediate (/Users/lchan/Projects/felicity/node_modules/lab/lib/runner.js:647:36)
      at processImmediate [as _immediateCallback] (timers.js:383:17)

  86) Binary should return a buffer of minimum size:

      Buffer.alloc is not a function

      at Object.internals.string.internals.number.internals.binary.schema [as binary] (/Users/lchan/Projects/felicity/lib/helpers.js:323:110)
      at /Users/lchan/Projects/felicity/test/value_generator_tests.js:546:40
      at Immediate._onImmediate (/Users/lchan/Projects/felicity/node_modules/lab/lib/runner.js:647:36)
      at processImmediate [as _immediateCallback] (timers.js:383:17)

  87) Binary should return a buffer of maximum size:

      Buffer.alloc is not a function

      at Object.internals.string.internals.number.internals.binary.schema [as binary] (/Users/lchan/Projects/felicity/lib/helpers.js:323:110)
      at /Users/lchan/Projects/felicity/test/value_generator_tests.js:554:40
      at Immediate._onImmediate (/Users/lchan/Projects/felicity/node_modules/lab/lib/runner.js:647:36)
      at processImmediate [as _immediateCallback] (timers.js:383:17)

  88) Binary should return a buffer of specified size:

      Buffer.alloc is not a function

      at Object.internals.string.internals.number.internals.binary.schema [as binary] (/Users/lchan/Projects/felicity/lib/helpers.js:323:110)
      at /Users/lchan/Projects/felicity/test/value_generator_tests.js:562:40
      at Immediate._onImmediate (/Users/lchan/Projects/felicity/node_modules/lab/lib/runner.js:647:36)
      at processImmediate [as _immediateCallback] (timers.js:383:17)

  89) Binary should return a buffer of size between min and max:

      Buffer.alloc is not a function

      at Object.internals.string.internals.number.internals.binary.schema [as binary] (/Users/lchan/Projects/felicity/lib/helpers.js:323:110)
      at /Users/lchan/Projects/felicity/test/value_generator_tests.js:571:40
      at Immediate._onImmediate (/Users/lchan/Projects/felicity/node_modules/lab/lib/runner.js:647:36)
      at processImmediate [as _immediateCallback] (timers.js:383:17)

  122) Object should return an object with specified keys:

      Buffer.alloc is not a function

      at Object.internals.string.internals.number.internals.binary.schema [as binary] (/Users/lchan/Projects/felicity/lib/helpers.js:323:110)
      at /Users/lchan/Projects/felicity/lib/helpers.js:499:147
      at Array.forEach (native)
      at Object.internals.string.internals.number.internals.binary.internals.date.internals.array.internals.object.Object.keys.forEach.schemaDescription.rules.forEach.schemaDescription.dependencies.forEach.peers.forEach [as object] (/Users/lchan/Projects/felicity/lib/helpers.js:483:126)
      at /Users/lchan/Projects/felicity/test/value_generator_tests.js:960:40
      at Immediate._onImmediate (/Users/lchan/Projects/felicity/node_modules/lab/lib/runner.js:647:36)
      at processImmediate [as _immediateCallback] (timers.js:383:17)


8 of 131 tests failed
Test duration: 389 ms
Assertions count: 1575 (verbosity: 12.02)
No global variable leaks detected
Coverage: 99.87% (1/754)
lib/helpers.js missing coverage on line(s): 329
Code coverage below threshold: 99.87 < 100
Linting results: No issues

npm ERR! Test failed.  See above for more details.
```

--- a/README.md
+++ b/README.md
@@ -132,14 +132,6 @@ Getting yourself setup and bootstrapped is easy.  Use the following commands aft
 npm install && npm test
 ```
 
-Felicity makes use of the Node.js `Buffer.alloc` function instead of the unsafe and deprecated `Buffer` constructor.
-As a result, one of the following minimum versions of [Node.js](//nodejs.org/en/download/releases/) is required:
-
-- v4.5.0+
-- v5.10.0+
-- v6.0.0+
-- v7.0.0+
-
 ## Joi features not yet supported
 
 Some Joi schema options are not yet fully supported. Most unsupported features should not cause errors, but may be disregarded by Felicity or may result in behavior other than that documented in the Joi api.

